WcaOnRails/app/models/person.rb

```diff
@@ -11,6 +11,13 @@ class Person < ActiveRecord::Base
 
   scope :current, -> { where(subId: 1) }
 
+  scope :in_region, lambda { |region|
```

Comment:
I'd like to check if it's a continent id or not and decide to use `Continent.c_all_by_id[region]&.countries&.map(&:id) ` or just `region`.